deploy/README.md

```
@@ -41,8 +41,38 @@ cd ../partner-onboarder
+     echo "Installing datashare"
+     helm -n $NS install datashare-inji mosip/datashare \
+     -f datashare-values.yaml \
+     --version $DATASHARE_CHART_VERSION
+     ```
+
+2. **Configure Mimoto for offline credentials:**
+   - Ensure that `qr_code_type` is set to `"EmbeddedVC"` in your `mimoto-issuers-config.json` file
+   - This embeds the entire Verifiable Credential in the QR code for offline verification
+
+3. **Run the modified install script:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)